src/sketch.js

```diff
@@ -2,21 +2,68 @@
 * DEFINITIONS
 */
 
+// sound controller container
+let sound = false;
+
 // perlin noise terrain container
 let terrain;
 
 // soundtrack p5.SoundFile container
 let soundtrack;
 
+/**
+ * I draw this sketch sound indicator.
+ * 
+ * Returns:
+ *  undefined.
+ */
+function drawSoundIndicator()
+{
+    // push transformation matrix
+    push();
+
+    // translate to bottom right corner
+    translate((WIDTH / 2) - 50, (HEIGHT / 2) - 50);
+
+    // set indicator transparency
+    sound ? tint(255, 100) : tint(255, 255);
+   
+    // draw image
+    image(keyboard, 0, 0, 50, 50);
+    
+    // pop transformation matrix
+    pop();
+}
+
 /**
 * p5.js preload callback.
 *
 * Returns:
 *   undefined;
 */
-function preload() {
-  // load soundtrack song
-  soundtrack = loadSound('assets/audio/da_beat-nr1.mp3');
+function preload() 
+{  
+    // load soundtrack song
+    soundtrack = loadSound('assets/audio/da_beat-nr1.mp3');
+
+    // load keyboard on image
+    keyboard = loadImage('assets/images/keyboard.png');
+}
+
+/**
+ * p5.js mouse clicked callback.
+ * 
+ * Returns:
+ *  undefined.
+ */
+function mouseClicked()
+{
+    // soundtrack icon was clicked: update soundtrack state
+    if(mouseX > WIDTH - 50  && mouseX < WIDTH && mouseY > HEIGHT - 50 && mouseY < HEIGHT)
+    {
+        sound = !sound;
+        sound ? soundtrack.loop() : soundtrack.stop();
+    }
```

Comment:
As we still haven't implemented buttons properly, why not make the whole viewport toggle this action? Codewise would be cleaner than hardcoding the icon area.

Reply:
As the webcam icon will be coming today too, we decided to keep it this way.